creating_packages/define_abi_compatibility.rst

```diff
@@ -20,7 +20,7 @@ When this package is installed by a *conanfile.txt*, another package *conanfile.
 
 .. code-block:: bash
 
-    $ conan install MyLib/1.0@user/channel -s arch=x86_64 -s ...
+    $ conan install . MyLib/1.0@user/channel -s arch=x86_64 -s ...
```

Comment:
is it `$ conan install MyLib/1.0@user/channel` or `$ conan install .` ?

Reply:
True, you either install the path "." or the package "MyLib/1.0@user/channel"